### PR TITLE
Fix make install bug

### PR DESCRIPTION
**Pull request by roystgnr**
This fixes the problem I hit from #218

We'll probably need to rebase #226 on this.

I think "put everything in share/examples" is the most standard way to go, but I could be talked out of that; I'm willing to settle for any install behavior which does not include "error and die".

### CODE REVIEW

File: examples/Makefile.am [hunk 1]
```diff
@@ -32,7 +27,9 @@ pkginclude_HEADERS =
 #
 # CPP TESTS
 #
+antioch_initdir = $(prefix)/share/examples/antioch_init
 antioch_init_SOURCES = antioch_init.C
+antioch_init_DATA    = ${antioch_init_SOURCES}
 
 #Define examples to actually be run
 TESTS  =
```
Comment (dmcdougall) [inline examples/Makefile.am]: Nuke this and the associated lines below it.  This doesn't belong to the test suite.

File: examples/Makefile.am [hunk 2]
```diff
@@ -32,7 +27,9 @@ pkginclude_HEADERS =
 #
 # CPP TESTS
 #
```
Comment (dmcdougall) [inline examples/Makefile.am]: Can also nuke these three comments.  They're just misleading.

File: examples/Makefile.am (hunk 1 above)
Comment (roystgnr) [inline examples/Makefile.am]: Leaving them.  More tests is better than fewer, even if we're pretty sure they're redundant.  We probably don't want to establish the *expectation* that examples also be regression tests, but we should leave open the *option*.

File: examples/Makefile.am (hunk 2 above)
Comment (roystgnr) [inline examples/Makefile.am]: Will do.

File: examples/Makefile.am (hunk 1 above)
Comment (dmcdougall) [inline examples/Makefile.am]: Makes sense.

**Comment by dmcdougall** [comment]
Thanks @roystgnr, sorry for the hiccup.

**Pull request merged**